src/allotropy/parsers/moldev_softmax_pro/softmax_pro_parser.py

```diff
@@ -1,16 +1,15 @@
 from typing import Any
 
 from allotropy.named_file_contents import NamedFileContents
-from allotropy.parsers.lines_reader import LinesReader
+from allotropy.parsers.lines_reader import CsvReader
 from allotropy.parsers.moldev_softmax_pro.softmax_pro_structure import Data
 from allotropy.parsers.vendor_parser import VendorParser
 
 
 class SoftmaxproParser(VendorParser):
     def to_allotrope(self, named_file_contents: NamedFileContents) -> Any:
-        contents = named_file_contents.contents
-        lines_reader = LinesReader(contents)
-        data = Data.create(lines_reader)
+        reader = CsvReader(named_file_contents.contents, encoding=None)
```

Comment:
@alejandro-salgado , this changed functionality: passing `encoding=None` means we no longer assume UTF-8 and let chardet detect the encoding. Was this intended?